src/components/my-widgets-scores.scss

```diff
@@ -10,6 +10,12 @@
 	overflow-y: auto;
 	padding-bottom: 1em;
 
+	&.limited_because_student {
```

Comment:
I'm assuming this was related to the `is_student` prop you're passing to `MyWidgetsScores`, but the wires didn't quite get hooked up.